Users/Install.rst

```diff
@@ -106,6 +106,13 @@ folder:
 
     $ virtualenv venv
 
+.. note::
+
+    If you have both Python 3 and Python 2 installed, use
+    ``virtualenv venv -p $(which python3)``.
+    This creates an isolated Python 3 environment called venv
```

Comment:
btw change this `called venv`.

Reply:
@newbazz I have changed it to:
 Python3 environment named ``venv`` 
 Does it seem fine??